### DISCUSSION TITLE
Option to Drop Accuracy Requirement After Opting Out of Leaderboard

### DISCUSSION THREAD

**Discussion by squeakyboots** [monkeytypegame/monkeytype#4744] [Ideas] [answered]
I'm using a chording device that results in MT recognizing low accuracy despite my accuracy being 100%. I understand there are requirements around accuracy for leaderboard purposes. I also understand there is no intention of supporting chording or steno on the leaderboards.

#3849 was raised (and implemented) to opt out for steno/chording users, but the accuracy requirement persists.

Can the opt out feature be expanded to drop the accuracy requirement to save results? I'd still like to know my PBs and stats apart from the leaderboard.

**Comment by Miodec** [comment, marked as answer]
I could probably lower it to 50% when opted out - would that work?

**Reply by squeakyboots** [reply]
Yes, it seems like my accuracy while chording usually sits in the high 60s (sometimes even clears the 75% mark), but bottoms around 60% so 50% should give a nice buffer. Thanks for being open to it!

**Reply by Miodec** [reply]
Added. Should be live later today.
